openapi.yml

```diff
@@ -0,0 +1,576 @@
+openapi: 3.1.0
+info:
+  title: Secure message v2
+  description: An updated service for sending and recieving secure messages
+  version: 1.0.0
+servers:
+  - url: https://localhost:5051
+    description: The path used by the secure message service when port forwarded through kubectl (5051:8080)
+tags:
+  - name: Messages
+    description: Message endpoints
+  - name: Threads
+    description: Thread endpoints
+  - name: Existing
+    description: Existing secure message functionality
+  - name: New
+    description: New to secure message v2
```

Comment:
I've added the `Existing` and `New` tags just for visibility which can be removed in future